Source/Utils/Average.cpp

```diff
@@ -27,3 +27,99 @@ Average::ToCellCenter ( MultiFab& mf_out,
                      } );
     }
 }
+
+void
+Average::CoarsenAndInterpolateLoop ( MultiFab& mf_cp,
+                                     const MultiFab& mf_fp,
+                                     const int dcomp,
+                                     const int scomp,
+                                     const int ncomp,
+                                     const IntVect ratio )
+{
+    // Staggerings of input fine MultiFab and output coarse MultiFab
+    const IntVect stag_fp = mf_fp.boxArray().ixType().ixType();
```

Comment:
```suggestion
    const IntVect stag_fp = mf_fp.boxArray().ixType().toIntVect();
    const IntVect stag_cp = mf_cp.boxArray().ixType().toIntVect();
```

Reply:
Thanks for this suggestion, I'll change this to be compliant with the new convention adopted with #895.